src/menu.cpp

```diff
@@ -1021,15 +1035,39 @@ class ControlsMenu final : public MenuBase
 
 const ControlsMenu::KeySettings ControlsMenu::c_key_settings[]=
 {
-	{ "Move Forward"	, SettingsKeys::key_forward	, KeyCode::W },
-	{ "Move Backward"	, SettingsKeys::key_backward	, KeyCode::S },
-	{ "Strafe Left"		, SettingsKeys::key_step_left	, KeyCode::A },
-	{ "Strafe Right"	, SettingsKeys::key_step_right	, KeyCode::D },
-	{ "Turn Left"		, SettingsKeys::key_turn_left	, KeyCode::Left },
-	{ "Turn Right"		, SettingsKeys::key_turn_right	, KeyCode::Right },
-	{ "Look Up"		, SettingsKeys::key_look_up	, KeyCode::Up },
-	{ "Look Down"		, SettingsKeys::key_look_down	, KeyCode::Down },
-	{ "Jump"		, SettingsKeys::key_jump	, KeyCode::Space },
+	/* essential controls */
+	{ "Move Forward"	, SettingsKeys::key_forward	  , KeyCode::W },
+	{ "Move Backward"	, SettingsKeys::key_backward	  , KeyCode::S },
+	{ "Strafe Left"		, SettingsKeys::key_step_left	  , KeyCode::A },
+	{ "Strafe Right"	, SettingsKeys::key_step_right	  , KeyCode::D },
+	{ "Speed Up"            , SettingsKeys::key_speed_up      , KeyCode::LeftShift },
+	{ "Always Run"          , SettingsKeys::key_always_run    , KeyCode::CapsLock },
+	{ "Jump"		, SettingsKeys::key_jump	  , KeyCode::Mouse2 },
+	{ "Fire"                , SettingsKeys::key_fire	  , KeyCode::Mouse1 },
+	/* original game and no mouse wheel user backward compatibility */
+	{ "Change Weapon"	, SettingsKeys::key_weapon_change , KeyCode::Mouse3 },
+	/* additional controls introduced in PanzerChasm with mouse wheel support */
+	{ "Next Weapon"         , SettingsKeys::key_weapon_next   , KeyCode::MouseWheelDown },
+	{ "Prev Weapon"     , SettingsKeys::key_weapon_prev   , KeyCode::MouseWheelUp },
+	{ "Menu"                , SettingsKeys::key_menu          , KeyCode::Escape },
```

Comment:
I think menu key (and also console key) should not be configurable. Who really needs this?